src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -22,7 +22,7 @@
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
     <IsPackable>false</IsPackable>
 
-    <GenerateResourceUsePreserializedResources>true</GenerateResourceUsePreserializedResources>
+    <GenerateResourceUsePreserializedResources Condition="'$(MSBuildRuntimeType)' == 'Core'">true</GenerateResourceUsePreserializedResources>
```

Comment:
So we decided not to create a menu item, but I wanted to leave the vsct file in place, and in particular this little condition, and the package reference bump above, which took me way too long to discover but were the secret to a working vsct file. My thinking is that if and when we do want to put a menu somewhere, it will be much easier having _something_ in place.

Reply:
Totally agree 👍